### ISSUE TITLE
Xades-bes

### ISSUE THREAD

**Issue by antonioanacletopt** [etobella/python-xades#13]
does is support xades-bes ?? is yes, how?? I tried but complains about the digest what is strange

**Comment by kubakrakowiak** [comment]
@IdeaSolutionsOnline Any ideas?

**Comment by antonioanacletopt** [comment]
Yes, I just removed the policy part because it does not belong to bes, be careful with the transforms and so, and it will work

**Comment by franfcunha** [comment]
Dear @IdeaSolutionsOnline,

How did you manage to remove the policy and, simultaneously, to have values in both xades:SigningCertificate and xades:SignaturePolicyIdentifier fields that are mandatory in the cape verde e-fatura?

Whenever I remove the policy part I get errors about these values being missing, and if I include a simple "ImpliedPolicy" the system throws an error that says that the digital signature must be of the type BES (Basic Electronic Signature)

Thank you!
